### ISSUE TITLE
Support for custom event loggers

### ISSUE THREAD

**Issue by jdai8** [bazelbuild/intellij#6433]
### Description of the bug:

I'd like to implement a custom event logger that sends data about sync / build times to an internal metrics service.

It looks like currently there is some existing code around [EventLoggingService](https://github.com/bazelbuild/intellij/blob/b86b9e45a02765b9835a635c712f120f66ddf287/base/src/com/google/idea/blaze/base/logging/EventLoggingService.java#L35) and [EventLogger](https://github.com/bazelbuild/intellij/blob/b86b9e45a02765b9835a635c712f120f66ddf287/base/src/com/google/idea/blaze/base/logging/EventLogger.java#L23) that might allow me to do this. However, I'm confused whether this is a supported feature or not.

There is the `com.google.idea.blaze.EventLogger` extension point, but it doesn't seem to be actually called from anywhere.

I could also implement the `EventLoggingService` interface, but the comment on that class suggests it's meant to just forward events to `EventLogger`. So I'm not sure if this is the correct path or not.

I'm opening this as a bug because the code looks like it tries to support this.

### Which category does this issue belong to?

Intellij

### What's the simplest, easiest way to reproduce this bug? Please provide a minimal example if possible.

N/A

### Which Intellij IDE are you using? Please provide the specific version.

IntelliJ Ultimate 2024.1

### What programming languages and tools are you using? Please provide specific versions.

Java

### What Bazel plugin version are you using?

N/A

**Comment by tpasternak** [comment]
Yeah, so it seems to be unused. Btw quite recently google commited an "intellijext" service for handling these logs here https://github.com/bazelbuild/intellij/commit/3dcb25108628d264da2e933e6231fbeaec8ff706 but I don't know what's the status of that. 

If it would work, you could even try to handle the logs in an external application. 

@mai93 do you know what's the status of this?
